admin/grants.sql

```diff
@@ -684,22 +684,30 @@ GRANT USAGE, MONITOR
 
 -- Grant schema privileges for SYNAPSE_AGGREGATE schema
 -- SYNAPSE_DATA_WAREHOUSE
-GRANT USAGE, MONITOR
-    ON SCHEMA SYNAPSE_DATA_WAREHOUSE.SYNAPSE_AGGREGATE
-    TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE.SYNAPSE_AGGREGATE_ALL_ADMIN;
+GRANT OWNERSHIP
+	ON SCHEMA SYNAPSE_DATA_WAREHOUSE.SYNAPSE_AGGREGATE
+	TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE.SYNAPSE_AGGREGATE_ALL_ADMIN;
 GRANT USAGE, MONITOR
     ON SCHEMA SYNAPSE_DATA_WAREHOUSE.SYNAPSE_AGGREGATE
     TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE.SYNAPSE_AGGREGATE_ALL_ANALYST;
 GRANT USAGE, MONITOR
     ON SCHEMA SYNAPSE_DATA_WAREHOUSE.SYNAPSE_AGGREGATE
     TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE.SYNAPSE_AGGREGATE_ALL_DEVELOPER;
+GRANT OWNERSHIP
+	ON ALL TABLES
+	IN SCHEMA SYNAPSE_DATA_WAREHOUSE.SYNAPSE_AGGREGATE
+	TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE.SYNAPSE_AGGREGATE_ALL_ADMIN;
 -- SYNAPSE_DATA_WAREHOUSE_DEV
-GRANT USAGE, MONITOR
+GRANT OWNERSHIP
     ON SCHEMA SYNAPSE_DATA_WAREHOUSE_DEV.SYNAPSE_AGGREGATE
     TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE_DEV.SYNAPSE_AGGREGATE_ALL_ADMIN;
 GRANT USAGE, MONITOR
     ON SCHEMA SYNAPSE_DATA_WAREHOUSE_DEV.SYNAPSE_AGGREGATE
     TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE_DEV.SYNAPSE_AGGREGATE_ALL_ANALYST;
 GRANT USAGE, MONITOR
     ON SCHEMA SYNAPSE_DATA_WAREHOUSE_DEV.SYNAPSE_AGGREGATE
-    TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE_DEV.SYNAPSE_AGGREGATE_ALL_DEVELOPER;
\ No newline at end of file
+    TO DATABASE ROLE SYNAPSE_DATA_WAREHOUSE_DEV.SYNAPSE_AGGREGATE_ALL_DEVELOPER;
+GRANT OWNERSHIP
+	ON ALL TABLES
```

Comment:
do you know if ALL TABLES means all table types, including dynamic tables? I couldn't find anything in the docs and tried testing it on Snowsight but it didn't seem to change anything 🤔

Reply:
I tried ALL TABLES and it only transferred ownership of tables, not dynamic tables.